.github/scripts/filter_changed_files_website.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+trigger_ci=false
+
+# If we are on main we want to always run the tests
+if [[ $GITHUB_REF_NAME == "main" ]]; then
+	trigger_ci=true
+	echo "On main, running CI: $trigger_ci"
+	echo "trigger-ci=$trigger_ci" >>"$GITHUB_OUTPUT"
+	exit 0
+fi
+
+# Get the list of changed files
+files_to_check=$(git diff --name-only origin/main)
+
+# Define the directories to check
+included_directories=("website" "packages/flight-icons/catalog.json")
```

Comment:
This would run website CI if any website file changed or if the underlying icon sprite changes (tested on the icon docs page). Do we need to also run the tests if components themselves change?